datafiles/manager.py

```diff
@@ -33,31 +33,28 @@ def all(self) -> Iterator[HasDatafile]:
         for filename in iglob(splatted):
             log.debug(f'Found matching path: {filename}')
             results = parse(pattern, filename)
-            fields = dataclasses.fields(self.model)
-            args = list(results.named.values())
-            args += [Missing] * (len(fields) - len(args))
-            yield self.model(*args)
+            yield self.get(*results.named.values())
 
-    def get_or_none(self, *args, **kwargs) -> Optional[HasDatafile]:
-        original_manual = self.model.Meta.datafile_manual
-
-        self.model.Meta.datafile_manual = True
```

Comment:
I don't know why this was there and abruptly removed it, so maybe it's worth putting it back in the new version?

Reply:
Alright, got it, this is the reason why the CI fails.